suite-native/tokens/src/tokensSelectors.ts

```diff
@@ -146,85 +145,24 @@ export const selectAnyOfTokensIsKnown = (
     return result;
 };
 
-const isNotZeroAmountTransfer = (tokenTransfer: TokenTransfer) =>
-    tokenTransfer.amount !== '' && tokenTransfer.amount !== '0';
-
-/** Is not a transaction with zero amount and no internal transfers. */
-const isNotEmptyTransaction = (transaction: WalletAccountTransaction) =>
-    transaction.amount !== '0' ||
-    (G.isArray(transaction.internalTransfers) && A.isNotEmpty(transaction.internalTransfers));
-
-const isTransactionWithTokenTransfers = (transaction: WalletAccountTransaction) =>
-    G.isArray(transaction.tokens) && A.isNotEmpty(transaction.tokens);
-
-const selectAccountTransactionsWithTokensWithFiatRates = createMemoizedSelector(
-    [
-        selectAccountTransactions,
-        selectTokenDefinitions,
-        (_state, _accountKey: AccountKey, areTokenOnlyTransactionsIncluded: boolean) =>
-            areTokenOnlyTransactionsIncluded,
-    ],
-    (
-        transactions,
-        tokenDefinitions,
-        areTokenOnlyTransactionsIncluded,
-    ): WalletAccountTransaction[] => {
-        return pipe(
+export const selectAccountTransactionsWithTokensWithFiatRates = createMemoizedSelector(
```

Comment:
Can we drop fiat rates from name? It is not valid anymore right?

Reply:
32c38f50ee